medcat-trainer/webapp/api/api/models.py

```diff
@@ -458,15 +458,24 @@ class Meta:
                                              'if a model pack is used for the project')
     relations = models.ManyToManyField('Relation', blank=True, default=None,
                                        help_text='Relations that will be available for this project')
+    use_model_service = models.BooleanField(default=False,
```

Comment:
Is there a need for both the boolean and the URL? Couoldn't we just assume that if there's a URL, it needs to be used?